src/schema/types/product.graphql

```diff
@@ -8,6 +8,6 @@ type Product {
 }
 
 extend type Query {
-  getProducts: [Product!]!
-  getProductById(id: ID!) : Product!
+  products: [Product!]!
+  product(id: ID!) : Product!
```

Comment:
The result of this query may be empty (non-existent ID).

Reply:
yes, you're right